src/main.tsx

```diff
@@ -5,6 +5,9 @@ import App from './App';
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
   <React.StrictMode>
-    <App />
+    <App
+      applicationId="AE8F7EEA-4555-4F86-AD8B-5E0BD86BFE67"
+      botId="khan-academy-bot"
```

Comment:
Maybe not commit these

Reply:
Yeah agreed. I moved them to env file. https://github.com/sendbird/chat-ai-widget/pull/4/commits/5f210df21365292662f05c006b4e81e819018084